xapian-core/cluster/cluster.cc

```diff
@@ -110,7 +114,7 @@ TermListGroup::add_document(const Document &document) {
 }
 
 void
-TermListGroup::add_documents(DocumentSource &docs) {
+TermListGroup::add_documents(MSetDocumentSource docs) {
```

Comment:
Sure. As you mentioned, I just changed that to get rid of the abstraction a while.